### DISCUSSION TITLE
bigger "Notes" width within the app

### DISCUSSION THREAD

**Discussion by obuno** [usememos/memos#2183] [General]
Hi there, thanks a bunch form Memos, has much of what I've been after.

I'm currently questioning myself if it would be possible to set a bigger width of the displayed notes on the main page?
![image](https://github.com/usememos/memos/assets/32042396/dbbb17fb-a178-41b9-9963-9e194bba7c53)

Thanks a lot,
Regards,
o.

**Comment by obuno** [comment]
Anyone ??

**Comment by obuno** [comment]
Okay, try harder style =) 

Adding these within the "Additional style" under Settings > System helped me at getting closer to what I want:

```
.w-auto {
  width: max-content;
}

.max-w-6xl {
  max-width: 108rem;
}
```

![image](https://github.com/usememos/memos/assets/32042396/e3c069d5-5f2a-4858-89df-b89491717fd5)

Hope this might help others.

Cheers,
O.

**Reply by jon-f-novabackup** [reply]
Thanks for the suggestion, it definately allows more text to be viewable in each memo, but depending on the resolution size (like the max width of the browser) of the web page that is being viewed for Memos the part that you mentioned ".max-w-6xl" may need to be changed to ".max-w-5xl" or ".max-w-4xl", because for me if I leave it at ".max-w-6xl" as stated, it cuts off the left toolbar too much and runs into the memo contents like this:

![image](https://github.com/usememos/memos/assets/50121594/ebdfc744-f040-4c99-a0af-a16472fd7fb1)

With ".max-w-5xl" in place it looks better, as far as allowing for about 20% greater width for memo contents in the same width size web browser window, like this:

![image](https://github.com/usememos/memos/assets/50121594/e960a068-f508-4d36-b1c2-2a00ade2d06e)

And this is the original regular width, without specifying any "Additional style" settings, which has about a 20% less width size for memo contents, in the same width size web browser window as all the rest of my examples here:

![image](https://github.com/usememos/memos/assets/50121594/4c7686a8-08a4-481a-a25f-37214744b83c)

It may be preferable on mobile web browsers though to keep the "Additional style" using the default settings, but at least this gives the end user the ability to control the sizing which is nice to have. It appears that this is a global system setting that would affect all users.

**Reply by VaultBoy88** [reply]
This does not work in 2025.

**Comment by themormansound** [comment]
I would also like to know whats changed and why it doesn't work in 2025?

**Reply by peterska** [reply]
try the following custom css:
```
.w-auto {
  width: max-content;
}

@layer theme {
    :root,:host {
        --container-xs: 30rem;
        --container-sm: 36rem;
        --container-md: 42rem;
        --container-lg: 48rem;
        --container-xl: 54rem;
        --container-2xl: 63rem;
        --container-3xl: 72rem;
        --container-5xl: 96rem;
       }
}
```
